package.json

```diff
@@ -16,5 +17,10 @@
   "repository": {
     "type": "git",
     "url": "git+https://github.com/vote/enip-backend.git"
+  },
+  "husky": {
```

Comment:
omg look at husky's github, it's like pre-commit.com crossed with twitter.com/stella2020woof